fargate-eventbridge-serverless/README.md

```diff
@@ -0,0 +1,82 @@
+# Leveraging Fargate for Scheduled Jobs
+
+This pattern contains a sample cloudformation template to deploy a scheduled AWS Fargate task using EventBridge running on an Amazon Elastic Container Service (ECS) cluster. The docker image is pushed to Amazon Elastic Container Registry (ECR) using cloudformation template without having to pre-push the image to ECR or another container library. The serverless framework plugin for fargate, `serverless-fargate` is used to deploy tasks on ECS Cluster. This task runs every 10th minute of the hour and a simple file containing JSON is put in S3 bucket. 
+
+Important: this application uses various AWS services and there are costs associated with these services after the Free Tier usage - please see the [AWS Pricing page](https://aws.amazon.com/pricing/) for details. You are responsible for any AWS costs incurred. No warranty is implied in this example.
+
+## Requirements
+
+* [Create an AWS account](https://portal.aws.amazon.com/gp/aws/developer/registration/index.html) if you do not already have one and log in. The IAM user that you use must have sufficient permissions to make necessary AWS service calls and manage AWS resources.
+* [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) installed and configured
+* [Git CLI](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) installed
+* [NodeJS](https://nodejs.org/en/download/) (LTS version) installed
+* [Serverless Framework CLI](https://www.serverless.com/framework/docs/getting-started) installed
+* [Docker Desktop](https://docs.docker.com/desktop/) installed
+
+## Deployment Instructions
+
+1. Create a new directory, navigate to that directory in a terminal and clone the GitHub repository:
+
+    ``` sh
+    git clone https://github.com/aws-samples/serverless-patterns
+    ```
+
+1. Change directory to the fargate-eventbridge pattern directory:
+
+    ``` sh
+    cd serverless-patterns/fargate-eventbridge-serverless
+    ```
+
+1. Open Docker Desktop and keep it running in the background.
+
+1. From the command line, use Serverless Framework to deploy the AWS resources for the pattern as specified in the serverless.yml file:
```

Comment:
need to install serverless-fargate before running this. Please add to the instructions

Reply:
The image would be built in Docker, so there's no need to install the package internally here.